app/controllers/AlertController.java

```diff
@@ -220,4 +318,5 @@ private AlertController(final int maxLimit, final AlertRepository alertRepositor
     private static final String NAGIOS_EXTENSION_NOTIFY_KEY = "notify";
     private static final String NAGIOS_EXTENSION_MAX_CHECK_ATTEMPTS_KEY = "max_check_attempts";
     private static final String NAGIOS_EXTENSION_FRESHNESS_THRESHOLD_KEY = "freshness_threshold";
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
You should use ObjectMapperFactory from commons here. Since you are not modifying the instance use .getInstance (as opposed to .createInstance()).